### ISSUE TITLE
Up to date limit per person

### ISSUE THREAD

**Issue by Krakaw** [big-neon/bn-api#800]
In the cart polling add the current ticket count of upcoming events for the user. This way when a user reaches their `limit_per_person` it will notify them instantly instead of multiple pings to the server / failure by trying.

> One thing that would be really helpful in the API is to get more caller-specific data in some instances. For example: Right now, in order to determine how many tickets a user can add to their cart I have to have A) an up-to-date ticket type response with its `limit_per_person` and `available` counts, B) an up-to-date list of the customer's already-purchased tickets, so i can see how many they already have, C) an up-to-date cart so i can see how many of that ticket type are in the cart
> 
> for ergonomics and fewer server requests, it would be helpful (for both success AND error responses) to get back the maximum number of tickets that the user is presently allowed to have in their cart. that way i don't have to constantly be pinging the server. for the extreme case: imagine a scenario where a very popular event is selling out very quickly. the available number of tickets is rapidly decreasing, and we need to keep making extra api calls just to see what the new limit is.
> 
> tl;dr every time i add a ticket to the cart (or fail at adding a ticket to the cart) it'd be great to get back "here's the max number this user can have in their cart, according to our latest calculations")
>
